backend/services/office_hours/ticket.py

```diff
@@ -0,0 +1,660 @@
+        # 1. If student, cannot leave feedback. Any staff member can close and leave feedback on any ticket.
+        if current_user_section_member_entity.member_role == RosterRole.STUDENT:
+            raise PermissionError(
+                f"Student's Do Not Have Permission to Give Feedback for Ticket id={oh_ticket.id}"
+            )
+
+        # 2. Check if Ticket Is Closed
+        if ticket_entity.state != TicketState.CLOSED:
+            raise PermissionError(
+                f"Ticket is Not Closed. Cannot Give Feedback for Ticket id={oh_ticket.id}"
+            )
+
+        # Update fields and commit changed entity
+        ticket_entity.have_concerns = oh_ticket.have_concerns
+        ticket_entity.caller_notes = oh_ticket.caller_notes
+
+        self._session.commit()
+
+        # Return details model
+        return ticket_entity.to_details_model()
+
+    def update_ticket_description(
+        self, subject: User, oh_ticket: OfficeHoursTicketPartial
+    ) -> OfficeHoursTicket:
+        """Update the description of an office hours ticket.
+
+        Args:
+            subject (User): The user attempting to update the ticket description.
+            oh_ticket (OfficeHoursTicketPartial): Partial ticket data including the new description.
+
+        Returns:
+            OfficeHoursTicket: Updated ticket details after the description has been updated.
+
+        Raises:
+            Exception: If the description data is missing or None.
+            ResourceNotFoundException: If the ticket with the provided ID does not exist.
+            PermissionError: If the user attempting to update is not the creator of the ticket
+                            or if the ticket is not in the QUEUED state.
+        """
+        # Check Feedback Fields Are Not None
+        if oh_ticket.description is None:
+            raise Exception("Missing Data To Update Ticket Description")
+
+        # Query Ticket
+        ticket_entity = self._session.get(OfficeHoursTicketEntity, oh_ticket.id)
+
+        if ticket_entity is None:
+            raise ResourceNotFoundException(f"Cannot Find Ticket id={oh_ticket.id}")
+
+        # USER PERMISSIONS:
+
+        # 1a. Fetch Office Hours Section - Needed To Determine if User Membership
+        oh_section_entity = self._get_office_hours_section_by_oh_event_id(
+            ticket_entity.oh_event_id
+        )
+
+        # 1b. Check Current User Section Membership
+        current_user_section_member_entity = self._check_user_section_membership(
+            subject.id, oh_section_entity.id
+        )
+
+        # 2. CASE: Only Creator(s) of Ticket Can Update
+        section_user_created_tickets: list[OfficeHoursTicketEntity] = (
+            current_user_section_member_entity.created_oh_tickets
+        )
+
+        # True if Current Ticket Exists In User's Created Ticket List
+        is_subject_creator: bool = any(
+            ticket.id == ticket_entity.id for ticket in section_user_created_tickets
+        )
```

Comment:
Wait, I did not know this existed lol - some of the other comments I said for rewiting code might be able to just use `any()`!